resources/views/component/offer.blade.php

```diff
@@ -0,0 +1,33 @@
+{{--
```

Comment:
Internally we call "offer" as "flight". Lets keep it consistent. In user facing documentation lets use "flight offer"